README.md

```diff
@@ -150,6 +150,25 @@ sentry_upload_sourcemap(
 )
 ```
 
+The `sourcemap` argument also accepts an array of files for uploading multiple
+source maps or bundle files simultaneously. This is required at least when
+using Hermes with React Native on Android.
+
+```ruby
+sentry_upload_sourcemap(
+  api_key: '...',
+  auth_token: '...',
+  org_slug: '...',
+  project_slug: '...',
+  version: '...',
+  app_identifier: '...', # pass in the bundle_identifer of your app
+  build: '...', # Optionally pass in the build number of your app
+  dist: '...', # optional distribution of the release usually the buildnumber
+  sourcemap: ['index.android.bundle', 'index.android.bundle.map'], # sourcemaps to upload
```

Comment:
We could also say that this can be a string to an individual file or an array with multiple files here.

Reply:
Simplified and tried to word it clearly but concisely.